### PR TITLE
Update prettier

### PR DESCRIPTION
**Pull request by koddsson**
## What I did

1. Updated prettier
2. Ran prettier

### CODE REVIEW

**Comment by koddsson** [comment]
Not sure why the action is reported red. Looks like everything is passing.

File: .changeset/light-llamas-remember.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"providence-analytics": patch
+"@lion/ui": patch
+---
+
+Reformatted code with version 3 of prettier
```
Comment (koddsson) [inline .changeset/light-llamas-remember.md:6]: Should this even be a version bump? We just formatted the code but didn't change anything. What do you think?

Comment (gerjanvangeest) [reply .changeset/light-llamas-remember.md:6]: No, that is not needed i guess

File: docs/fundamentals/systems/overlays/use-cases.md [hunk 2]
```diff
@@ -132,14 +132,12 @@ render() {
 or declaratively in your template with the `.config` property
 
 ```html
-<demo-el-using-overlaymixin .config=${{ ...withModalDialogConfig() }}>
+<demo-el-using-overlaymixin .config="${{" ...withModalDialogConfig() }}>
```
Comment (gerjanvangeest) [inline docs/fundamentals/systems/overlays/use-cases.md:135]: This one goes wrong

File: docs/fundamentals/systems/overlays/use-cases.md [hunk 3]
```diff
@@ -132,14 +132,12 @@ render() {
...
   <button slot="invoker">Click me to open the overlay!</button>
   <div slot="content" class="demo-overlay">
     Hello! You can close this notification here:
-    <button
-      @click=${e => e.target.dispatchEvent(new Event('close-overlay', { bubbles: true }))}
-    >
-      ⨯
+    <button @click="${e" ="">
```
Comment (gerjanvangeest) [inline docs/fundamentals/systems/overlays/use-cases.md:139]: Same here

File: docs/fundamentals/tools/helpers/action-logger.md [hunk 4]
```diff
@@ -42,11 +42,10 @@ This connects the logger element to the trigger.
 
 ```html
 <div>To log: <code>Hello, World!</code></div>
-<button
-  @click=${e => {
-    e.target.parentElement.querySelector('#logger-${uid}').log('Hello, World!')
-  }}
->Click this button</button>
+<button @click="${e" ="">
```
Comment (gerjanvangeest) [inline docs/fundamentals/tools/helpers/action-logger.md:45]: This one goes wrong

**Review by gerjanvangeest — COMMENTED** [review]
@koddsson thanks for the contribution :)

File: docs/fundamentals/systems/overlays/use-cases.md (hunk 2 above)
Comment (koddsson) [reply docs/fundamentals/systems/overlays/use-cases.md]: Hehe this is funny. Prettier tries to parse these lit templates as HTML which of course fails. I changed lit snippets to JSX which seems to fix it but it's a shame that prettier doesn't seem to have Lit template support.

**Review by tlouisse — APPROVED** [review]

**Comment by tlouisse** [comment]
Great, thanks 💪

**Pull request merged**